mysql-test/suite/rpl/t/rpl_column_compression_async_rpl.test

```diff
@@ -0,0 +1,32 @@
+#
+# PS-7806 : Column compression breaks async replication on PS
+#
+
+--source include/not_group_replication_plugin.inc
+--source include/master-slave.inc
+
+sync_slave_with_master;
+
+--source include/rpl_connection_master.inc
```

Comment:
You can remove these lines. The rationale is 
1. `--source include/master-slave.inc` by default leaves the connection on master, so there is no need to change the connection to master.
2. As we haven't done anything in the test, there is nothing to sync.